dev/tests/integration/framework/Magento/TestFramework/TestCase/AbstractController.php

```diff
@@ -268,14 +269,21 @@ protected function getCookieMessages($messageType = null)
     {
         /** @var $cookieManager CookieManagerInterface */
         $cookieManager = $this->_objectManager->get(CookieManagerInterface::class);
+
+        /** @var $jsonSerializer \Magento\Framework\Serialize\Serializer\Json */
+        $jsonSerializer = $this->_objectManager->get(\Magento\Framework\Serialize\Serializer\Json::class);
```

Comment:
Now I was not sure if I should use the class `\Magento\Framework\Serialize\Serializer\Json` here or just `json_*` but I thought since this was using the cookie interface it should also use the serializer. Happy to make a change if it is thought best to do something else.